src/s2wasm.h

```diff
@@ -467,6 +470,7 @@ class S2WasmBuilder {
       else if (match("ident")) skipToEOL();
       else if (match("section")) parseToplevelSection();
       else if (match("align") || match("p2align")) skipToEOL();
+      else if (match("import_global")) parseImportGlobal();
```

Comment:
Yes, which more succinctly expresses the intent, and also fits nicely with the assembler directives on the previous line that we also just skip.